### PR TITLE
JStorm-runner: Implementation of processing timer

### PR DESCRIPTION
**Pull request by bastiliu**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by bastiliu** [comment]
@peihe

File: runners/jstorm/src/main/java/org/apache/beam/runners/jstorm/translation/TimerServiceImpl.java [hunk 1]
```diff
@@ -139,26 +126,47 @@ public void addWatermarkHold(String namespace, Instant watermarkHold) {
     }
   }
 
+  private PriorityQueue<TimerInternals.TimerData> getTimerQueue(TimeDomain timeDomain) {
```
Comment (peihe) [inline runners/jstorm/src/main/java/org/apache/beam/runners/jstorm/translation/TimerServiceImpl.java]: Note for me: move it to the bottom.

File: runners/jstorm/src/main/java/org/apache/beam/runners/jstorm/translation/ExecutorsBolt.java [hunk 2]
```diff
@@ -44,6 +46,8 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+
```
Comment (peihe) [inline runners/jstorm/src/main/java/org/apache/beam/runners/jstorm/translation/ExecutorsBolt.java]: Note for me: will remove during merging

**Comment by peihe** [comment]
Merged

**Pull request closed without merging**